.github/ISSUE_TEMPLATE/Bug report form.yml

```diff
@@ -0,0 +1,127 @@
+name: Report a problem/bug
+title: '[Problem/Bug]: '
+description: File a report for something not working.
```

Comment:
nit:
I think "File a report for something not working" sounds a little unnatural.

Would be good to just be "Report a problem" or "Submit a bug report."